backend/cfg/cfg_irc.ml

```diff
@@ -605,21 +466,12 @@ let run : Cfg_with_liveness.t -> Cfg_with_liveness.t =
     | false -> ()
     | true -> Cfg_with_liveness.invalidate_liveness cfg_with_liveness));
   main ~round:1 state cfg_with_liveness;
-  (* note: slots need to be updated before prologue removal *)
-  if irc_debug
-  then
-    Array.iteri (State.num_stack_slots state)
-      ~f:(fun reg_class num_stack_slots ->
-        log ~indent:1 "stack_slots[%d]=%d" reg_class num_stack_slots);
-  update_stack_slots cfg_with_layout
-    ~num_stack_slots:(State.num_stack_slots state);
-  remove_prologue_if_not_required cfg_with_layout;
-  update_register_locations ();
-  update_live_fields cfg_with_layout
-    (Cfg_with_liveness.liveness cfg_with_liveness);
-  if irc_debug && irc_invariants
-  then (
-    log ~indent:0 "postcondition";
-    postcondition cfg_with_layout ~allow_stack_operands:true);
-  Array.iter all_precolored_regs ~f:(fun reg -> reg.Reg.degree <- 0);
+  Cfg_regalloc_rewrite.postlude
+    (module State)
+    (module Utils)
+    state
+    ~f:(fun () ->
+      update_register_locations ();
+      Array.iter all_precolored_regs ~f:(fun reg -> reg.Reg.degree <- 0))
```

Comment:
To be clear, the next round will be for another function, so
indeed this is unnecessary. However, `Cfg_irc_state.make`
will set these fields to "infinite"; my intent was to restore
the value to a "sane" value from a non-IRC perspective.
(But to be honest, I don't think anyone will look at the
degrees after allocation, so that is not a practical concern.)